### ISSUE TITLE
Screen readers can read content outside the date picker modal

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4075]
## Area or Component
Sign up form - personal details - DOB
(This is likely happening for other date entries on forms, like the person form)

## Quick Summary
Screen readers can read content outside the modal dialogue (in this case, the date picker popup).

### How to Replicate
1. Navigate to https://dev.simplereport.gov/app/sign-up/identity-verification (you'll probably need to create a new org first)
2. Open the date picker
3. Attempt to screen read outside the picker

## Accessibility Violation
Able to browse outside modal with screen reader: https://cdc-devtools.dequecloud.com/rule-help/en/reading-order-browse-outside-modal

## Action Requested
All other entries on the page should become `aria-hidden: true`. (Note: this seems to be an issue for all modals on our site...will probably need to do some research to see if React or USWDS has a generic way to do this)

## Screenshots (optional)
![Screen Shot 2022-07-26 at 3 11 41 PM](https://user-images.githubusercontent.com/80282552/181121141-35d5a336-796a-4339-82b5-1db520d98007.png)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=84:84

**Comment by BobanL** [comment]
So i've been trying to use screen reader (my first time really), and on the page mentioned (page specifically, I can't seem to leave with the modal still open as mentioned? Is it possible, it has been fixed on that page? 
 
On the `Add Patient` page though, I believe I am able to hop out while the date picker is still active.
<img width="1057" alt="image" src="https://user-images.githubusercontent.com/10108172/186961553-d9362258-bc2a-4537-b48e-aae5e805c2fa.png">


The date picker on the signup verification uses the react-uswds component where as the other one just uses an input field. So if that's the case maybe making them all use the uswds component would be sufficient.

**Comment by emmastephenson** [comment]
@BobanL I'm also not familiar with screen readers, this issue came up when I was doing a guided test as part of Deque. Unfortunately, that guided test says this is still an issue 🤔 Their guidance is the following:

`Add aria-hidden="true" attribute to all wrapping elements within <body> that do not contain the modal. You do not need to add aria-hidden="true" to <script> and <style> elements as screen readers do not read content in those tags. Since some assistive technologies do not yet fully support the aria-modal attribute, we recommend using other techniques to hide content outside of a dialog from screen readers.`

This datepicker does have `aria-modal` set, which is why I'm guessing you weren't able to navigate outside it using the screen reader - your screenreader/browser combo likely has support for it. The one on the person form likely doesn't have that `aria-modal` set, which is why you were able to read outside.
I think changing all the datepickers to the react-uswds component is still a good plan, but unfortunately it seems like you'll also need to set all other page elements to `aria-hidden=true` :/
